docs/UserGuide.md

```diff
@@ -271,20 +287,18 @@ Example of usage:
 You are running low on food in these categories:
 1. VEGETABLE quantity: 0 out of 500
 2. FRUIT quantity: 0 out of 500
-3. MEAT quantity: 200 out of 500
-4. SEAFOOD quantity: 1 out of 500
-5. EGG quantity: 0 out of 500
-6. DAIRY quantity: 0 out of 500
-7. BEVERAGE quantity: 0 out of 500
-8. COOKED_DISH quantity: 0 out of 500
-9. READY_TO_EAT quantity: 1 out of 500
-10. FROZEN quantity: 0 out of 500
-11. OTHER quantity: 0 out of 500
+3. SEAFOOD quantity: 0 out of 500
+4. EGG quantity: 0 out of 500
+5. DAIRY quantity: 0 out of 500
+6. BEVERAGE quantity: 0 out of 500
+7. COOKED_DISH quantity: 0 out of 500
+8. FROZEN quantity: 0 out of 500
+9. OTHER quantity: 0 out of 500
 ```
 
 ### Modify the minimum quantity limits: `setlimit`
 
-Changes the minimum quantity limit for a specific food category.
+Changes the minimum quantity limit for a specific food category. To set remove a limit for a category, set its limit to -1.
```

Comment:
Would it be possible to change the limit to 0?

Reply:
modified